src/Microsoft.DotNet.Darc/src/DarcLib/Helpers/HttpRequestManager.cs

```diff
@@ -58,6 +58,12 @@ public async Task<HttpResponseMessage> ExecuteAsync(int retryCount = 15)
                         return response;
                     }
                 }
+                catch (HttpRequestException reqEx) when (reqEx.Message.Contains("404 (Not Found)"))
+                {
+                    _logger.LogError($"There was a 'Not Found' exception executing method '{_method}' against URI '{_requestUri}'. " +
```

Comment:
This should be wrapped with if(_logFailure).  There are cases where a 404 is expected.

Reply:
As in just log and throw if `_logFailure` is `true` or just log if this is `true` and throw regardless?

Reply:
@mmitche ?

Reply:
log if true and throw regardless.  The logfailure flag helps reduce noise in "known" failure scenarios (like expected 404)

Reply:
Thanks! Addressed